packages/dev/docs/src/Layout.js

```diff
@@ -485,7 +485,6 @@ function Nav({currentPageName, pages}) {
             <details key={section.title} open={section.isActive}>
               <summary style={{fontWeight: 'bold'}}>
                 <ChevronRight size="S" /> {section.title}
-                {section.title === 'Components' && <VersionBadge version={Object.values(section.pages)[0][0].preRelease} style={{marginLeft: 'auto', fontWeight: 'normal'}} />}
```

Comment:
Since we added Autocomplete (the first component alphabetically) all of RAC was getting marked as alpha again 😂